### ISSUE TITLE
!storepass doesn't work at all on OFTC

### ISSUE THREAD

**Issue by grischard** [matrix-org/matrix-appservice-irc#1483]
**Describe the bug**

It's impossible to store a nickserv password or certificate on OFTC.

OFTC doesn't support using server password to identify to services, which means currently there's no way to automatically reidentify after connection blips between the oftc bridge and their network. OFTC does support cert auth.

**To Reproduce**
1. Connect to OFTC. Register your nickname with NickServ.
2. Tell @oftc-irc:​matrix.org to `!storepass foo`
3. `!reconnect`

**Expected behaviour**

Nickname gets authenticated on NickServ (through certfp) on reconnection.

**Actual behaviour**

Nickname doesn't get authenticated, making it impossible to rejoin some channels. No error messages.

**Additional context**

There was #459 which got closed even though the problem still exists.

#1128 to add certfp was abandoned. #1331 which was done instead doesn't fix the issue - authentication fails even with `!username`.

#747 tracks certfp support, but doesn't make clear that authentication is currently completely broken on OFTC.

**Comment by Mikaela** [comment]
I would close this as a duplicate of #747 or #714. Alternatively a documentation issue.

I think the current strategy may be to just wait for OFTC to migrate to Solanum and likely gain SASL support like the rest of IRCv3 capable networks.

**Comment by grischard** [comment]
Hi! I definitely agree with you on it being a documentation issue - it was very confusing, and I was pretty sure I was doing something wrong.

**Comment by TheEvilSkeleton** [comment]
I tried on LiberaChat and it doesn't seem to work on my side as well. I did `IDENTIFY $NICK $PASSWORD` with `@NickServ:libera.chat`, and then `!storepass $NICK:$PASSWORD` and lastly `!storepass $PASSWORD` with `@appservice:libera.chat`, but after a couple of days I get kicked out in some rooms.

**Comment by Mikaela** [comment]
Yes, it's documented a bit unclearly in https://matrix-org.github.io/matrix-appservice-irc/latest/usage.html

**Comment by z3ntu** [comment]
So there's no workaround for OFTC at the moment?

**Comment by z3ntu** [comment]
From a user perspective (which I am) I really don't care whose fault it is, I just need to know whether it works and if yes how. And if it doesn't work or if it does work with some workaround then this should be documented of course - but from what I get from your comment it doesn't work, yes?

**Comment by Mikaela** [comment]
You are correct. I opened an issue to OFTC side https://github.com/oftc/oftc-hybrid/issues/69, which either they can fix or this could be closed in favour of the previously mentioned #747 or #714, I don't think tracking this duplicate separately is helpful towards anyone.

**Comment by Half-Shot** [comment]
Hi there, it's been some time but given I had a few minutes of energy left in the day, https://github.com/matrix-org/matrix-appservice-irc/pull/1757 now exists to solve this problem.

**Comment by funderscore1** [comment]
Can't the bridge just gain support for automatically sending commands to nickserv?
